### PR TITLE
sqsh 2.5.16.1 (new formula)

### PR DESCRIPTION
**Pull request by meissnem**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----
`sqsh` was placed in the boneyard on 9/16/2016 because it failed to build. This change fixes the build failure.

I'm unsure of the proper way to reinstate a formula from the boneyard. If this request is accepted, do I create a pull-request against homebrew-boneyard to remove `sqsh` there?

### CODE REVIEW

**Comment by woodruffw** [comment]
One minor audit failure:

```
Error: 1 problem in 1 formula
sqsh:
  * New formulae should not use `deprecated_option`.
```

File: Formula/sqsh.rb [hunk 1]
```diff
@@ -0,0 +1,102 @@
+class Sqsh < Formula
+  desc "Sybase Shell"
+  homepage "https://sourceforge.net/projects/sqsh/"
+  url "https://downloads.sourceforge.net/project/sqsh/sqsh/sqsh-2.5/sqsh-2.5.16.1.tgz"
+  sha256 "d6641f365ace60225fc0fa48f82b9dbed77a4e506a0e497eb6889e096b8320f2"
+
+  deprecated_option "enable-x" => "with-x11"
+
+  depends_on :x11 => :optional
```
Comment (MikeMcQuaid) [inline Formula/sqsh.rb]: Remove this option and the `deprecated_option` above.

File: Formula/sqsh.rb [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+  depends_on "freetds"
+  depends_on "readline"
+
+  # this patch fixes detection of freetds being installed, it was reported
+  # upstream via email and should be fixed in the next release
+  patch :DATA
```
Comment (MikeMcQuaid) [inline Formula/sqsh.rb]: What date and what email? I'm tempted to say we wait for a new release before we re-add this.

File: Formula/sqsh.rb [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+    readline = Formula["readline"]
+    ENV["LIBDIRS"] = readline.opt_lib
+    ENV["INCDIRS"] = readline.opt_include
+
+    if build.with? "x11"
+      args << "--with-x"
+      args << "--x-libraries=#{MacOS::X11.lib}"
+      args << "--x-includes=#{MacOS::X11.include}"
+    end
```
Comment (MikeMcQuaid) [inline Formula/sqsh.rb]: Remove all this.

**Review by MikeMcQuaid — CHANGES_REQUESTED** [review]

**Comment by fxcoudert** [comment]
Closing for now, please reopen when the comments are addressed (and a new release is out).

**Comment by developish** [comment]
@meissnem can you address those comment and reopen? I'd find it valuable.

**Comment by woodruffw** [comment]
@developish We probably won't un-boneyard this in either case, since upstream hasn't made a release in ~3 years and we don't want to end up fielding support for their software.

If you'd like to use this, I'd suggest adding it to a [personal tap](https://github.com/Homebrew/brew/blob/master/docs/How-to-Create-and-Maintain-a-Tap.md) with the patch and changes above.

**Pull request closed without merging**